### PR TITLE
📝 Add RFC for Organization

### PR DESCRIPTION
**Pull request by dankolbman**
RFC write up for the proposed Organization entity.

### CODE REVIEW

File: docs/rfcs/0001_organizations.md
Comment (dankolbman) [inline docs/rfcs/0001_organizations.md:3]: This will be 'approved' once it merges, until then, it's still in review.

File: docs/rfcs/0001_organizations.md
Comment (znatty22) [inline docs/rfcs/0001_organizations.md:13]: > An Organization encompasses two important entities: studies and users. 

Is storage backends not included here because it will be added in another RFC?

File: docs/rfcs/0001_organizations.md
Comment (znatty22) [inline docs/rfcs/0001_organizations.md:49]: Hmm maybe only superusers (I think that's what our current Admin group represents) should be able to create a new organization?

File: docs/rfcs/0001_organizations.md
Comment (znatty22) [inline docs/rfcs/0001_organizations.md:97]: I wonder if it would still be useful to have a superuser group that would have admin privs for all organizations.

**Review by znatty22 — COMMENTED** [review]
There isn't any mention of app routes changing or use of subdomains for organization so I'm assuming we're keeping those the same? So for example, the route for a file will be `/studies/SD_ME0WME0W/documents/SF_00000000` vs something like `/kids-first/studies/SD_ME0WME0W/documents/SF_00000000` or `kids-first.kf-ui-data-tracker.org/studies/SD_ME0WME0W/documents/SF_00000000`

File: docs/rfcs/0001_organizations.md
Comment (znatty22) [reply docs/rfcs/0001_organizations.md:13]: Also wondering if we should support templates per organization or per study?

Comment (dankolbman) [reply docs/rfcs/0001_organizations.md:13]: Storage backends, templates, and whatever else are future propositions whereas studies and users are things we have to deal with at implementation time for Organizations. Those details can be spec'd in their own docs.

File: docs/rfcs/0001_organizations.md
Comment (dankolbman) [reply docs/rfcs/0001_organizations.md:49]: Perhaps to start, but the idea is that anyone may join and create an organization.

File: docs/rfcs/0001_organizations.md
Comment (dankolbman) [reply docs/rfcs/0001_organizations.md:97]: Yeah the permissions are going to need their own re-design.

**Review by znatty22 — APPROVED** [review]
👥 ✅

**Pull request merged**
